### PR TITLE
Run dev appserver from exploded-war directory

### PR DESCRIPTION
**Pull request by loosebazooka**
- Also fix appengine-web.xml parsing in tests

### CODE REVIEW

**Comment by loosebazooka** [comment]
@meltsufin

**Comment by loosebazooka** [comment]
Fixes #48

File: src/test/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPluginTest.java [hunk 1]
```diff
@@ -50,7 +53,8 @@ public void setUp() throws IOException {
 
     Path webInf = testProjectDir.getRoot().toPath().resolve("src/main/webapp/WEB-INF");
     Files.createDirectories(webInf);
-    Files.createFile(webInf.resolve("appengine-web.xml"));
+    File appengineWebXml = Files.createFile(webInf.resolve("appengine-web.xml")).toFile();
+    com.google.common.io.Files.append("<appengine-web-app/>", appengineWebXml, Charsets.UTF_8);
```
Comment (meltsufin) [inline src/test/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPluginTest.java]: you can use JDK's Files.write instead here

**Comment by meltsufin** [comment]
LGTM, except for that one comment.

**Pull request merged**